ui-participant/src/util/loggingUtils.ts

```diff
@@ -39,7 +39,7 @@ export const logError = (detail: ErrorEventDetail, stackTrace: string) => {
   Api.log({
     eventType: 'ERROR',
     eventName: 'jserror',
-    eventDetail: JSON.stringify(detail),
+    eventDetail: `${JSON.stringify(detail)}\n${window.location.href}`,
```

Comment:
Should this be its own attribute? Maybe `eventLocation`? As-is this might make it more difficult to categorize or filter error events.

Reply:
yeah, I think you're right -- I was too lazy to make it on this PR since it was just a quick one-off.  Can I take an IOU and I'll add that column when we add this error logging to the admin tool?  Then we can think about whether we also want to add a 'source' column for participant/admin/server

Reply:
works for me 👍